### PR TITLE
Fixed the null prefix in object name issue

### LINKED ISSUE #500: [Multicloud UI] Object names are prefixed with null when an object is uploaded on a fresh installation of SODA
**Issue/Feature Description:**
When an object is uploaded to multicloud backend on a fresh installation of SODA the object name is prefixed with null. 

**Why this issue to fixed / feature is needed(give scenarios or use cases):**
User is unable to delete the object until another object is first created which uploads with correct name.

**How to reproduce, in case of a bug:**
1. Install SODA
2. On a fresh installation register a cloud backend
2. Create a bucket
3. Upload an object

**Other Notes /  Environment Information: (Please give the env information, log link or any useful information for this issue)**
create-object-null-in-name

### PR DESCRIPTION
**Pull request by anvithks**
**What type of PR is this?**
/kind bug fix

**What this PR does / why we need it**:
This PR fixes the issue of null being prefixed in object name when an object is uploaded on a fresh installation of SODA.

**Which issue(s) this PR fixes**:
Fixes #500 

**Test Report Added?**:
/kind TESTED

**Test Report**:

**Special notes for your reviewer**:

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -307,7 +307,7 @@ export class AppComponent implements OnInit, AfterViewInit {
         window['startUpload'] = (selectFile, bucketId, uploadOptions,folderId, cb) => {
             window['isUpload'] = true;
             this.showPrompt =  true;
-            if(folderId !=""){
+            if(folderId !== null && folderId !== ""){
```
Comment (kumarashit) [inline src/app/app.component.ts:310]: Why to check not equal type?

Comment (anvithks) [reply src/app/app.component.ts:310]: It is a check whether the current location is inside a folder. If we create a folder then the folderId will have the folder name.
We check whether the folderId is not empty and not null.
On fresh installation of SODA, the variable that holds the folder name is not set and is equal to null. This was causing the object name to be prefixed with null as there was only a check on whether the folder name was not empty. null is not empty although they both solve to false.

Comment (anvithks) [reply src/app/app.component.ts:310]: On re-reading this I realise you were asking about the type check. 

The issue we have is that on first load of SODA Dashboard when we have not created a folder the object folderId is not initiated. When the value is checked it shows up as null. As mentioned in the second link below if it is undefined the type could be undefined and if it is null the type is object. 
Hence the type check.

More info [here](https://medium.com/javascript-in-plain-english/how-to-check-for-null-in-javascript-dffab64d8ed5#:~:text=Comparisons%20can%20be%20made%3A%20null,a%20value%20is%20not%20null%20.) and [here](https://stackoverflow.com/a/16204960/6303009)

Comment (kumarashit) [reply src/app/app.component.ts:310]: Ok

Comment (kumarashit) [reply src/app/app.component.ts:310]: Good so now the value and type both are checked

**Review by kumarashit — APPROVED** [review]
LGTM

File: src/app/app.component.ts (hunk 1 above)
Comment (anvithks) [reply src/app/app.component.ts:310]: Yes. that is correct.

**Review by skdwriting — APPROVED** [review]
LGTM

**Pull request merged**